docs/getting_started/hopsworksai/guides/eks_ecr_integration.rst

```diff
@@ -0,0 +1,327 @@
+    :figclass: align-center
+
+Click *Save rules* to save the updated rules to the security group.
+
+Step 5: Create a hopsworks cluster with EKS and ECR support
+-----------------------------------------------------------
+
+In Hopsworks.ai, select *Create cluster*. Choose the region of your EKS cluster, then click Next:
+
+.. _eks-hopsworks-create-cluster.png: ../../../_images/eks-hopsworks-create-cluster.png
+.. figure:: ../../../imgs/hopsworksai/eks-hopsworks-create-cluster.png
+    :alt: Create Hopsworks cluster
+    :target: `eks-hopsworks-create-cluster.png`_
+    :align: center
+    :figclass: align-center
+
+Choose your preferred SSH key to use with the cluster, then click Next:
+
+.. _eks-hopsworks-create-cluster-0.png: ../../../_images/eks-hopsworks-create-cluster-0.png
+.. figure:: ../../../imgs/hopsworksai/eks-hopsworks-create-cluster-0.png
+    :alt: Create Hopsworks cluster
+    :target: `eks-hopsworks-create-cluster-1.png`_
+    :align: center
+    :figclass: align-center
+
+Choose Enabled to enable the use Amazon EKS and ECR:
+
+.. _eks-hopsworks-create-cluster-1.png: ../../../_images/eks-hopsworks-create-cluster-1.png
+.. figure:: ../../../imgs/hopsworksai/eks-hopsworks-create-cluster-1.png
+    :alt: Create Hopsworks cluster
+    :target: `eks-hopsworks-create-cluster-1.png`_
+    :align: center
+    :figclass: align-center
+
+Add your EKS cluster name and update your AWS account id if you want to use another account for ECR, then click Next:
+
+.. _eks-hopsworks-create-cluster-2.png: ../../../_images/eks-hopsworks-create-cluster-2.png
+.. figure:: ../../../imgs/hopsworksai/eks-hopsworks-create-cluster-2.png
+    :alt: Create Hopsworks cluster
+    :target: `eks-hopsworks-create-cluster-2.png`_
+    :align: center
+    :figclass: align-center
+
+Choose the instance profile role that you have created in Step 2, then click Next:
+
+.. _eks-hopsworks-create-cluster-3.png: ../../../_images/eks-hopsworks-create-cluster-3.png
+.. figure:: ../../../imgs/hopsworksai/eks-hopsworks-create-cluster-3.png
+    :alt: Create Hopsworks cluster
+    :target: `eks-hopsworks-create-cluster-3.png`_
+    :align: center
+    :figclass: align-center
+
+
+Choose the VPC of your EKS cluster, then click Next:
+
+.. _eks-hopsworks-create-cluster-4.png: ../../../_images/eks-hopsworks-create-cluster-4.png
+.. figure:: ../../../imgs/hopsworksai/eks-hopsworks-create-cluster-4.png
+    :alt: Create Hopsworks cluster
+    :target: `eks-hopsworks-create-cluster-4.png`_
+    :align: center
+    :figclass: align-center
+
+Choose any of the subnets in the VPC, then click Next:
+
+.. _eks-hopsworks-create-cluster-5.png: ../../../_images/eks-hopsworks-create-cluster-5.png
+.. figure:: ../../../imgs/hopsworksai/eks-hopsworks-create-cluster-5.png
+    :alt: Create Hopsworks cluster
+    :target: `eks-hopsworks-create-cluster-5.png`_
+    :align: center
+    :figclass: align-center
+
+
+Choose the security group that you have updated in Step 4, then click Next:
```

Comment:
EKS creates three Security Groups and step 4 is far behind especially if you follow the instructions for the first time. Can you say something like Select the Security Group in the form of `eks-cluster-sg-CLUSTER-NAME` and **NOT** the ones for `ControlPlaneSecurity` or `ClusterSharedNode`. Or something similar ;) It's easy to mess it up here